_posts/2025-04-09-#31664.md

```diff
@@ -10,17 +10,96 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### Motivation and Background
 
+The current Fee Estimator in Bitcoin Core ([`CBlockPolicyEstimator`](https://github.com/bitcoin/bitcoin/blob/4c1906a500cacab385b09e780b54271b0addaf4b/src/policy/fees.h#L148)) used by the wallet and exposed to clients via RPC, has two key limitations that affect users:
 
-## Questions
+1. **Mempool unaware**: Does not consider the current mempool unconfirmed transactions when estimating fees.
+
+2. **Package unaware**: Only considers individual transactions, ignoring parent-child relationships.
+
+These limitations cause practical problems for:
+- Bitcoin Core wallet users (the majority now rely on third-party providers)
+- Lightning implementations: (e.g., C-lightning)
+- Bitcoin clients: ([e.g., BTCPay Server](https://github.com/btcpayserver/btcpayserver/pull/5490))
```

Comment:
Short descriptions instead of just names could help here. Also, why is this significant - should we just be really happy and delete the fee estimator from bitcoin core if there is a useful external one?